dts/bindings/timer/microchip,xec-rtos-timer.yaml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2019 Microchip Technology Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: Microchip RTOS timer
+version: 0.1
+
+description: >
+    This is a representation of the Microchip RTOS timer node
```

Comment:
Change the sentence to : This is a representation of the Microchip XEC RTOS timer node
just to make the sentence more specific